spring-cloud-skipper/spring-cloud-skipper-server/pom.xml

```diff
@@ -41,6 +41,11 @@
 		</dependencies>
 	</dependencyManagement>
 	<dependencies>
+		<dependency>
```

Comment:
Please revert - the file is the same as before other than "formatting" which equates to another file to review and resonate with.